co2-sensor/main.py

```diff
@@ -0,0 +1,33 @@
+import subprocess
+import re
+import time
+
+from prometheus_client import CollectorRegistry, Gauge, push_to_gateway
+
+PUSHGATEWAY_ADDRESS = "localhost:9091"
+
+
+def main():
+    for i in range(0, 100, 1):
+        co2_conce = get_co2_conce()
+        send_co2_conce(co2_conce)
+        print(i, "分経過: ", co2_conce)
```

Comment:
nits: printでコンマで並べると間に空白が入るので、気になるようであれば文字列で結合したほうが良いです。
f-stringで書くのが今風です。

```suggestion
        print(f"{i}分経過: {co2_conce}")
```

Reply:
対応しました。